pom.xml

```diff
@@ -78,7 +78,7 @@
         <commons.cli.version>1.3.1</commons.cli.version>
         <jackson-databind.version>2.11.2</jackson-databind.version>
         <jjwt.version>0.10.5</jjwt.version>
-        <ldaptive.version>1.2.3</ldaptive.version>
+        <ldaptive.version>1.2.4</ldaptive.version>
```

Comment:
This is the problem I highlighted [here](https://github.com/opensearch-project/opensearch-plugins/pull/119#discussion_r774627116) with the blanket replacement of numbers.